src/test/java/BaseTest.java

```diff
@@ -1,18 +1,54 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.firefox.FirefoxDriver;
 import org.openqa.selenium.safari.SafariDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Parameters;
+
+import java.time.Duration;
 
 
 public class BaseTest {
 
+    public static WebDriver driver;
+    public static WebDriverWait wait;
+
+
+
     @BeforeSuite
-    static void setupClass() {
+    @Parameters({"BaseURL"})
+    public static void setupClass(String BaseURL) {
         WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+       // String baseURL = "https://bbb.testpro.io/";
+        String url = BaseURL;
+        driver.get(url);
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+    }
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/224632481-a9b43303-847f-4b3c-89a2-ec5d3caedfb4.png)

**Hint**

1.

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >

<suite name="TestNG">

    <test name="Example Test" preserve-order="false">
      <parameter name="BaseURL" value="https://bbb.testpro.io/"></parameter>
        <classes>
           <class name="LoginTests"/>
           <class name="BaseTest"/>
            <class name="HomeWork19"/>
        </classes>
    </test>
</suite>
```
2.

 ```
   public static WebDriver driver;
    public static WebDriverWait wait;

    @BeforeSuite
    static void setupClass() {
        WebDriverManager.chromedriver().setup();
    }


    @BeforeMethod
    @Parameters({"BaseURL"})
    public static void setupClass(@Optional String BaseURL) {
        ChromeOptions options = new ChromeOptions();
        options.addArguments("--disable-notifications","--remote-allow-origins=*", "--incognito","--start-maximized");
        driver = new ChromeDriver(options);
        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
        String url = BaseURL;
        driver.get(url);

    }


```